forui/lib/src/widgets/breadcrumb.dart

```diff
@@ -207,24 +284,41 @@ class _CollapsedCrumbState extends State<_CollapsedCrumb> with SingleTickerProvi
   @override
   void initState() {
     super.initState();
-    controller = FPopoverController(vsync: this);
+    controller = widget.popoverController ?? FPopoverController(vsync: this);
```

Comment:
You need to add a the didUpdateWidget to handle if the widget controller changes. Dispose needs to change too.